plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -695,7 +695,44 @@ function OPDSBrowser:downloadFile(item, filename, remote_url)
     end
 end
 
-function OPDSBrowser:streamPages(item, remote_url, count)
+-- This function shows a dialog with input fields
+-- for entering information for Page streaming from.
+-- an input page number
+function OPDSBrowser:jumpToPage(viewer, count)
+    local input_dialog
+    input_dialog = InputDialog:new{
+        title = "Enter Page Number",
+        input = "",
+        input_type = "number",
+        input_hint = "Page # (1 - "..count..")",
```

Comment:
Use this one instead so it's greppable:
https://github.com/koreader/koreader/blob/ee35257f997ec3cc719d23212428d39b4173129d/frontend/ui/widget/keyvaluepage.lua#L390-L392

Reply:
Is this what you're looking for? `input_hint = "Page # ".."(" .. "1 - " .. count .. ")",`

Reply:
Apologies for being unclear. My objection is to the "Page #". Forget about the rest.